.travis.yml

```diff
@@ -1,62 +1,58 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - nightly
-  - hhvm-3.12
+    - 5.6
+    - 7.0
+    - 7.1
+    - nightly
+    - hhvm-3.12
 
 matrix:
-  allow_failures:
```

Comment:
> all our files are indented 4 spaces
All our files are PHP source files not a Travis config file

You can do the same here
https://github.com/getsentry/sentry/blob/master/.travis.yml 
https://github.com/getsentry/sentry/blob/master/src/sentry/api/authentication.py 

See? No difference